### ISSUE TITLE
Zoom into histogram

### ISSUE THREAD

**Issue by Telofy** [getguesstimate/guesstimate-app#967]
It would also be great if, in the expanded view, I could zoom into one segment of the x axis, because `=lognormal(0, 2)` for example is so flat and close to the x axis for most of the length that I can’t see much anyway, and I would much rather zoom into the [0, 5] or [0, 10] interval where there’s more going on. Thank you!

**Comment by OAGr** [comment]
Good point. I'll think about decent ways of doing this. Maybe it could be done in a non-fancy way.
